### PR TITLE
Bump org.apache.maven.plugins.maven-wrapper-plugin from 3.1.1 to 3.2.0

### PR DESCRIPTION
**Pull request by c00ler**

### CODE REVIEW

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#   https://www.apache.org/licenses/LICENSE-2.0
+#   http://www.apache.org/licenses/LICENSE-2.0
```
Comment (guylabs) [inline .mvn/wrapper/maven-wrapper.properties:9]: Why this change?

**Review by guylabs — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -194,6 +194,11 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-wrapper-plugin</artifactId>
+                    <version>3.2.0</version>
+                </plugin>
```
Comment (welandaz) [inline pom.xml:201]: Why do we need this plugin now?

**Review by welandaz — APPROVED** [review]

File: .mvn/wrapper/maven-wrapper.properties (hunk 1 above)
Comment (c00ler) [reply .mvn/wrapper/maven-wrapper.properties:9]: I don't know :) All the changes are produced by the wrapper goal. My guess is that it's some alignment.

Comment (c00ler) [reply .mvn/wrapper/maven-wrapper.properties:9]: @guylabs see here: https://github.com/apache/maven-wrapper/commit/8a1ded6cc4de5b2c9e238efe84ae122fdd8db050

File: pom.xml (hunk 2 above)
Comment (c00ler) [reply pom.xml:201]: It's not a plugin definition, it's a `<pluginManagement>`. We are overriding the version shipped with the maven and can use `mvn wrapper:wrapper` in the future. Otherwise, we would need always to use full plugin coordinates. 

For example in https://github.com/gradle/maven-build-scan-quickstart dependabot made the changes by specifying the full coordinates in the command line. If now one will execute `mvn wrapper:wrapper -Dmaven=3.9.0` using the latest maven 3.9.0, the wrapper version will be downgraded back to 3.1.1. See: https://scans.gradle.com/s/il4shfvrnbz7e/console-log?page=1#L8
